### PR TITLE
W0 Fixing postcourse github issues/bugs 

### LINKED ISSUE #256: W0D1 Exercise 8
From @rshpeley in #243

This exercise is meant to replicate exercise 7 but using numpy arrays in place of python lists. The solution for exercise 7 and its graph shows all 150 points are plotted. This solution plots 149 points and misses the first one. You'll see this visually in comparing the 2 graphs.

The recommended code changes increase the size of the v array by one to accommodate the last calculated update to v and slices v back to size before plotting:

```diff
 # set random number generator
 np.random.seed(2020)

 # initialize step_end, t_range, v and syn
 step_end = int(t_max / dt)
 t_range = np.linspace(0, t_max, num=step_end)
-v = el * np.ones(step_end)
+v = el * np.ones(step_end + 1)
 syn = i_mean * (1 + 0.1 * (t_max/dt) ** (0.5) * (2 * np.random.random(step_end) - 1))

 # loop for step_end - 1 steps
-for step in range(1, step_end):
+for step in range(step_end):

-  v[step] = v[step - 1] + (dt / tau) * (el - v[step - 1] + r * syn[step])
+  v[step + 1] = v[step] + (dt / tau) * (el - v[step] + r * syn[step])

 with plt.xkcd():
   # initialize the figure
@@ -20,5 +19,5 @@ with plt.xkcd():
   plt.xlabel('time (s)')
   plt.ylabel(r'$V_m$ (V)')

-  plt.plot(t_range, v, 'k.')
+  plt.plot(t_range, v[:-1], 'k.')
   plt.show()
```

### PR DESCRIPTION
**Pull request by JesseLivezey**
Closes #256, W0D1
No W0D2 tagged github issues to fix

### CODE REVIEW

**Comment by mwaskom** [comment]
@JesseLivezey does this fix the issue that #444 addresses?

**Comment by JesseLivezey** [comment]
@mwaskom No, missed that. I can add another commit soon.

**Comment by JesseLivezey** [comment]
@mwaskom this should be good to go.

File: tutorials/W0D1_PythonWorkshop1/W0D1_Tutorial1.ipynb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "id": "view-in-github"
    },
    "source": [
-    "<a href=\"https://colab.research.google.com/github/NeuromatchAcademy/course-content/blob/master/tutorials/W0D1_PythonWorkshop1/W0D1_Tutorial1.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+    "<a href=\"https://colab.research.google.com/github/NeuromatchAcademy/course-content/blob/w0-postcourse-bugfix/tutorials/W0D1_PythonWorkshop1/W0D1_Tutorial1.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
[the remark this answers is not in the document]
Comment (mwaskom) [reply tutorials/W0D1_PythonWorkshop1/W0D1_Tutorial1.ipynb:10]: Yes, but it can be fixed if we remember.

**Pull request merged**